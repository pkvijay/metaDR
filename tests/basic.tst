list /
create /ltest 123
create /ltest/a 456
create /ltest/b 789
list /ltest
remove /ltest/b
list /ltest
list /lmissing_key
list /
remove /ltest/a
remove /ltest
create BADPATH 123
create /ctest 123
create /ctest 123
create /ctest/b abc
create /cfoo/c def
remove /ctest/b
remove /ctest
remove BADPATH
create /rtest 123
remove /rtest
remove /rtest
remove /rfoo
create /rtest abc
create /rtest/b def
remove /rtest
remove /rtest/b
remove /rtest
get BADKEY
set BADKEY 123
create /gtest 123
get /gtest
set /gtest foo
get /gtest
get /gmissing_key
set /gmissing_key bar
remove /gtest
