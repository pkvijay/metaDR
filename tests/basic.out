CREATED
CREATED
CREATED
a
b
REMOVED
a
KEY NOT FOUND
ltest
REMOVED
REMOVED
MALFORMED KEY
CREATED
KEY ALREADY EXISTS
CREATED
NO PARENT
REMOVED
REMOVED
MALFORMED KEY
CREATED
REMOVED
KEY NOT FOUND
KEY NOT FOUND
CREATED
CREATED
HAS CHILDREN
REMOVED
REMOVED
MALFORMED KEY
MALFORMED KEY
CREATED
123
foo
KEY NOT FOUND
KEY NOT FOUND
REMOVED
